src/polygon.c

```diff
@@ -850,6 +852,51 @@ spherepoly_in(PG_FUNCTION_ARGS)
 	PG_RETURN_POINTER(poly);
 }
 
+Datum
+spherepoly_deg(PG_FUNCTION_ARGS)
+{
+	int          i, np;
```

Comment:
Fix the identation.

![image](https://github.com/postgrespro/pgsphere/assets/3510112/ff9656bd-75de-48a3-a202-b98532eb0c2a)